src/app/csc/csc.component.html

```diff
@@ -70,79 +65,62 @@ <h4 class="no_data">
         class="csc-projects"
         *ngIf="!dataLoading && filteredCscProjectsList.length > 0"
       >
-        <div class="d-none d-lg-block">
-          <ng2-smart-table
-            class="table table-striped"
-            [settings]="settings"
-            [source]="filteredCscProjectsList"
-          ></ng2-smart-table>
-        </div>
-      </div>
-    </div>
-    <!-- ********************< Mobile Cards*****************************************************-->
```

Comment:
I don't think we should be removing this for mobile functionality -- do an A/B between this branch and production to see why (in mobile emulation mode)

Reply:
Added back in! Thanks for catching this